starters/basic-starter/pages/[...slug].tsx

```diff
@@ -21,16 +25,57 @@ const CONTENT_TYPES = [
   'node--event',
   'node--person',
   'node--place',
+  'taxonomy_term--article_type',
+  'taxonomy_term--categories',
+  'taxonomy_term--event_type',
+  'taxonomy_term--person_type',
+  'taxonomy_term--place_type',
 ];
 
 interface NodePageProps extends LayoutProps {
```

Comment:
Hmm, `NodePageProps` sounds specific to nodes. I think we need some refactoring to make this generic so that it makes sense for both, taxonomy terms and nodes.

Reply:
Maybe we should convert some of the logic into components and then this would become "EntityPage" or something along the lines of that